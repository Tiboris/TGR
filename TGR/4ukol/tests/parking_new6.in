B01 6,6: 5
B02 1,1: 5
P01 3,3: 2
P02 1,2: 3
P03 6,8: 2
P04 4,6: 3
