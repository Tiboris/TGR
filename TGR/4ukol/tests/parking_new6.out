B02_1 P01_1 4
B02_2 P01_2 4
B02_3 P02_1 1
B02_4 P02_2 1
B02_5 P02_3 1
B01_1 P03_1 2
B01_2 P03_2 2
B01_3 P04_1 2
B01_4 P04_2 2
B01_5 P04_3 2
Celkem: 21
